Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM alpine:3.6
-# TODO: maybe we can use scratch again
+FROM scratch
```

Comment:
what are the benefits of scratch vs alpine? has it any tag?

Reply:
scratch does not have tags. Scratch is as a "keyword" to say "no base layer at all", so its the "minimal of the minimal" base image you can use to build your docker image

Reply:
ack